packages/cli/commands/lint.js

```diff
@@ -19,44 +20,47 @@ const i18nKey = 'cli.commands.lint';
 const { EXIT_CODES } = require('../lib/enums/exitCodes');
 
 exports.command = 'lint <path>';
-// Hiding since this command is still experimental
-exports.describe = null; //'Lint a file or folder for HubL syntax';
+exports.describe = 'Lint a file or folder for HubL errors and warnings';
 
 exports.handler = async options => {
-  const { path: lintPath } = options;
-
   await loadAndValidateOptions(options);
 
+  const { path: lintPath } = options;
   const accountId = getAccountId(options);
   const localPath = resolveLocalPath(lintPath);
+
+  trackCommandUsage('lint', null, accountId);
+
   const groupName = i18n(`${i18nKey}.groupName`, {
     path: localPath,
   });
-
-  trackCommandUsage('lint', null, accountId);
+  let errorCount = 0;
 
   logger.group(groupName);
-  let count = 0;
+
   try {
     await lint(accountId, localPath, result => {
-      count += printHublValidationResult(result);
+      errorCount = printHublValidationResult(result);
     });
   } catch (err) {
-    logger.groupEnd(groupName);
     logErrorInstance(err, { accountId });
     process.exit(EXIT_CODES.ERROR);
   }
-  logger.groupEnd(groupName);
-  logger.log(
-    i18n(`${i18nKey}.issuesFound`, {
-      count,
-    })
-  );
+
+  logger.groupEnd();
+  errorCount === 0
+    ? logger.log(i18n(`${i18nKey}.noIssuesFound`))
+    : logger.log(
+        i18n(`${i18nKey}.issuesFound`, {
+          errorCount,
+        })
+      );
```

Comment:
I found that when `0` was returned for the errorCount, the log would show ` issues found` instead of `0 issues found` -- even though `0` was being passed into the interpolator. I couldn't track down why that was happening, so I added in another key here for that scenario with `No issues found` string.

Reply:
This was due to a bug in i18n string interpolation - fixed it [here](https://github.com/HubSpot/hubspot-cli/pull/933/files)

Reply:
Thanks! I tried tracing that into the interpolator but I couldn't pinpoint that line as the issue -- I can revert this back -- it would be better to keep this simple